codemp/rd-rend2/tr_glsl_parse.cpp

```diff
@@ -41,7 +41,8 @@ Block *FindBlock( const char *name, Block *blocks, size_t numBlocks )
 	return nullptr;
 }
 
-#if defined(__clang__)
+// [M] strncpy_s is not present on linux and VS only function
+#ifndef WIN32
```

Comment:
I prefer to use the `defined` function - can this be changed to `#if !defined(_WIN32)` (notice the underscore prefix too)?